packages/server/src/oauth/token.ts

```diff
@@ -673,21 +674,3 @@ function verifyCode(challenge: string, method: string, verifier: string): boolea
 
   return false;
 }
-
-/**
- * Returns the base64-url-encoded SHA256 hash of the code.
- * The details around '+', '/', and '=' are important for compatibility.
- * See: https://auth0.com/docs/flows/call-your-api-using-the-authorization-code-flow-with-pkce
- * See: packages/client/src/crypto.ts
- * @param code - The input code.
- * @returns The base64-url-encoded SHA256 hash.
- */
-export function hashCode(code: string): string {
```

Comment:
I moved this from `oauth/token.ts` to `oauth/utils.ts` because it was causing circular import issues.